homework9_1.txt

```diff
@@ -0,0 +1,307 @@
+dr-xr-x---.   6 root root 4,0K ноя 29 00:38 root
+drwxr-xr-x.  24 root root  720 ноя 29 00:41 run
+lrwxrwxrwx.   1 root root    8 окт 23 19:06 sbin -> usr/sbin
+drwx------+   3 root root   19 окт 29 15:15 shares
+drwxr-xr-x.   2 root root    6 апр 11  2018 srv
+dr-xr-xr-x.  13 root root    0 ноя 29 00:41 sys
+drwxrwxrwt.  10 root root 4,0K ноя 29 00:42 tmp
+drwxr-xr-x.  13 root root  155 окт 23 19:06 usr
+drwxr-xr-x.  19 root root  267 окт 23 19:13 var
+
+# Монтирую:
+[root@localhost ~]# mount /dev/sdc1 /backup/
+
+[root@localhost ~]# ll /dev/sd*
+brw-rw----. 1 root disk 8,  0 ноя 29 00:41 /dev/sda
+brw-rw----. 1 root disk 8,  1 ноя 29 00:41 /dev/sda1
+brw-rw----. 1 root disk 8,  2 ноя 29 00:41 /dev/sda2
+brw-rw----. 1 root disk 8, 16 ноя 29 00:41 /dev/sdb
+brw-rw----. 1 root disk 8, 17 ноя 29 00:41 /dev/sdb1
+brw-rw----. 1 root disk 8, 18 ноя 29 00:41 /dev/sdb2
+brw-rw----. 1 root disk 8, 32 ноя 29 00:41 /dev/sdc
+brw-rw----. 1 root disk 8, 33 ноя 29 00:41 /dev/sdc1
+brw-rw----. 1 root disk 8, 34 ноя 29 00:41 /dev/sdc2
+
+[root@localhost ~]# df -h
+Файловая система        Размер Использовано  Дост Использовано% Cмонтировано в
+/dev/mapper/centos-root   6,2G         1,8G  4,4G           30% /
+devtmpfs                  485M            0  485M            0% /dev
+tmpfs                     496M            0  496M            0% /dev/shm
+tmpfs                     496M         512K  496M            1% /run
+tmpfs                     496M            0  496M            0% /sys/fs/cgroup
+/dev/sdb1                 976M         2,6M  907M            1% /archive
+/dev/sda1                1014M         159M  856M           16% /boot
+tmpfs                     100M            0  100M            0% /run/user/0
+/dev/sdc1                 2,0G          33M  2,0G            2% /backup
+
+[root@localhost ~]# blkid /dev/sdc1
+/dev/sdc1: UUID="d43b591e-59ca-4a24-a290-dda65be757fc" TYPE="xfs" PARTLABEL="Linux filesystem" PARTUUID="885ee88f-be0a-421f-a97d-8916bfc11ce7" 
+
+[root@localhost ~]# blkid /dev/sdc2
+/dev/sdc2: UUID="bb82b46f-9ef5-4972-8c29-2aa71ece057c" TYPE="swap" PARTUUID="d97ec4fe-79fc-4203-a06c-7e518f76e31e" 
+
+1.6. Configure the newly created swap space to be enabled at boot:
+
+[root@localhost ~]# vim /etc/fstab
+/*
+#
+# /etc/fstab
+# Created by anaconda on Tue Oct 23 19:06:23 2018
+#
+# Accessible filesystems, by reference, are maintained under '/dev/disk'
+# See man pages fstab(5), findfs(8), mount(8) and/or blkid(8) for more info
+#
+/dev/mapper/centos-root /                       xfs     defaults        0 0
+UUID=ea174e9b-e4a2-45be-8d22-ffb18f18b7f6 /boot                   xfs     defaults        0 0
+/dev/mapper/centos-swap swap                    swap    defaults        0 0
+
+UUID=46956463-a2ec-4705-94d1-d3bfb9d1d52a /archive                ext4    defaults        0 1 
+ 
+UUID=d43b591e-59ca-4a24-a290-dda65be757fc /backup       xfs     defaults 0 0
+UUID=bb82b46f-9ef5-4972-8c29-2aa71ece057c swap  swap    defaults        0 0 
+*/
+
```

Comment:
It would be nice to see here a command that will verify /etc/fstab has correct syntax before doing reboot

Reply:
mount -a 
The command will mount all the filesystems mentioned in the fstab.